### ISSUE TITLE
Focus bug when MyFinder is selected on a specific screen

### ISSUE THREAD

**Issue by FulytheFox** [mydockfinder/mydockfinder#416]
**Expected Behavior:**
When focusing a window on another screen, MyFinder should display the window name and settings.

**Current Behavior:**
It doesn't switch to the focused window on the other screen and displays the window name and settings of the last focused window on the screen where MyFinder is selected.

**Steps to Reproduce:**
1. Go to `Preferences... > MyFinder Look & Behaviour` and select a specific screen under `Select wich screens display Finder`.
2. Select a window that is on the selected screen.
3. Select a window that is on a different screen. MyFinder should not switch to the new window.

**Comment by mydockfinder** [comment]
Yes, I designed it this way, myfinder will only show the title of the same screen window




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "mydockfinder/mydockfinder-for-Win10-Win11"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月18日(星期四) 晚上9:50
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[mydockfinder/mydockfinder-for-Win10-Win11] Focus bug when MyFinder is selected on a specific screen (Issue #416)





 
Expected Behavior:
 When focusing a window on another screen, MyFinder should display the window name and settings.
 
Current Behavior:
 It doesn't switch to the focused window on the other screen and displays the window name and settings of the last focused window on the screen where MyFinder is selected.
 
Steps to Reproduce:
  
Go to Preferences... &gt; MyFinder Look &amp; Behaviour and select a specific screen under Select wich screens display Finder.
 
Select a window that is on the selected screen.
 
Select a window that is on a different screen. MyFinder should not switch to the new window.

**Comment by FulytheFox** [comment]
I understand your point, especially if you use it on all screens, it makes sense. However, it does confuse me sometimes when I focus a window that is on another screen and then have no settings on MyFinder.

How about an option on how MyFinder should behave in this situation?

**Comment by mydockfinder** [comment]
可以让myfinder在所有屏幕显示，在偏好设置-myfinder通用中，显示myfinder所在屏幕，选择所有屏幕
&nbsp;


№じ&nbsp;尐乄鈊~→☆
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年8月22日(星期一) 晚上7:52
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Focus bug when MyFinder is selected on a specific screen (Issue #416)






我理解你的观点，特别是如果你在所有的屏幕上使用，这是有道理的。然而，有时当我聚焦在另一个屏幕上的窗口，然后在MyFinder上没有设置时，我会感到困惑。

如何选择MyFinder在这种情况下的行为？

—
直接回复此邮件，在GitHub上查看，或取消订阅.
***@***.***与&gt;.
